src/CLICommand/cli_command.ts

```diff
@@ -62,7 +62,7 @@ function setCommanderCommand(commandDescriptor: CommandDescriptor, program: CliA
 				exitProgram(exitCode || 0);
 			});
 		} catch (e) {
-			console.error(`Error: ${e.message}`);
+			console.error(`Error: ${(e as Error).message}`);
```

Comment:
Might be safer to do a ternary check on the type and if it's not an Error then just string interpolate whatever bubbled out here.

Reply:
5e860af